components/api/src/main/java/com/hotels/styx/api/messages/CustomHttpResponseStatus.java

```diff
@@ -13,9 +13,7 @@
   See the License for the specific language governing permissions and
   limitations under the License.
  */
-package com.hotels.styx.api.netty;
-
-import com.hotels.styx.api.messages.HttpResponseStatus;
+package com.hotels.styx.api.messages;
 
 /**
  * Custom HTTP response status codes.
```

Comment:
We need to remove the `netty` package from the API. Can you raise another GitHub issue for this please?

Reply:
Raised #207